app/styles/templates/_login.scss

```diff
@@ -4,10 +4,46 @@
   }
 }
 
-.login-page {
-  @media #{$medium-only}{
-    margin-left: 3.5rem;
+.warning_empty{
+  padding-left: 1rem !important;
```

Comment:
All those `!important` are really needed ?

There are lots of `!important` in this file. Just verify it once and if not needed then remove. So many `important` creates mess later on. You know that already. So just confirming. :)